app/Filament/Resources/EmulatorResource/RelationManagers/SystemsRelationManager.php

```diff
@@ -55,10 +55,11 @@ public function table(Table $table): Table
 
             ])
             ->headerActions([
-
+                Tables\Actions\AttachAction::make()
+                    ->preloadRecordSelect(),
```

Comment:
Should this generate an audit log entry?

Reply:
It should indeed, updated in latest.